### PR TITLE
cfm: update to 0.6.0

### PR DESCRIPTION
**Pull request by WillEccles**
#### Description

[cfm](https://cfm.atinycact.us) is a simple and fast TUI file manager.

###### Type(s)

- [ ] bugfix
- [ ] enhancement
- [ ] security fix

###### Tested on

macOS 10.14.4 18E226
Xcode 10.3 10G8

###### Verification 
Have you

- [x] followed our [Commit Message Guidelines](https://trac.macports.org/wiki/CommitMessages)?
- [x] squashed and [minimized your commits](https://guide.macports.org/#project.github)?
- [x] checked that there aren't other open [pull requests](https://github.com/macports/macports-ports/pulls) for the same change?
- [x] referenced existing tickets on [Trac](https://trac.macports.org/wiki/Tickets) with full URL?

- [x] checked your Portfile with `port lint`?
- [x] tried existing tests with `sudo port test`?
- [x] tried a full install with `sudo port -vst install`?
- [x] tested basic functionality of all binary files?

### CODE REVIEW

**Comment by WillEccles** [comment]
Whoops, didn't actually mean to add those check-boxes, I was just messing with them and forgot to uncheck before submitting the PR.

**Comment by mf2k** [comment]
Can you please fix the case on your GitHub handle in the `maintainers` line? Apparently GitHub is confused about this and does not realize your are the maintainer of this port.

**Comment by WillEccles** [comment]
@mf2k what do you mean? I got assigned and pinged by the bot, is that not what should happen?

**Comment by mf2k** [comment]
It's not supposed to notify you because you are the submitter (and maintainer). Also the label says "maintainer: requires approval" instead of simply "maintainer".

**Comment by WillEccles** [comment]
Oh, I see what you mean. Should I remove the email and just leave `maintainers @willeccles` then?

**Comment by mf2k** [comment]
It's exactly right as is. Just replace "@willeccles" with "@WillEccles".

**Comment by WillEccles** [comment]
I presume you wanted the capital letters, which GitHub so kindly added for you. Can do.

**Comment by WillEccles** [comment]
Actually, to be honest, I would rather my username have lowercase letters on GitHub. I'm gonna look into that and see if it can be done.

**Comment by WillEccles** [comment]
I take that back, I'd rather not change all of my upstream URLs.

File: sysutils/cfm/Portfile [hunk 1]
```diff
@@ -3,18 +3,18 @@
...
-github.setup        willeccles cfm 0.5.5 v
+github.setup        willeccles cfm 0.6.0 v
 categories          sysutils
 platforms           darwin
 license             MPL-2
-maintainers         {eccles.dev:will @willeccles}
+maintainers         {eccles.dev:will @WillEccles}
 description         Cactus File Manager
 long_description    Cactus File Manager is a simple and fast TUI file manager
 homepage            https://github.com/willeccles/cfm
```
Comment (mf2k) [inline sysutils/cfm/Portfile]: The `homepage` line can be removed because that is the default one for the github portgroup.

Comment (WillEccles) [reply sysutils/cfm/Portfile]: Resolved in https://github.com/macports/macports-ports/pull/6508/commits/6d0fd9c6d89228cf9516a9cff7a59c579ff12d34

**Pull request merged**